sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_last_seen_v1/query.sql

```diff
@@ -39,11 +39,7 @@ _previous AS (
 )
 SELECT
   @submission_date AS submission_date,
-  IF(
-    _current.user_id IS NOT NULL,
-    _current,
-    _previous
-  ).* REPLACE ( --
+  IF(_current.user_id IS NOT NULL, _current, _previous).* REPLACE ( --
```

Comment:
I suspect some of these empty comments were added specifically to have the preceding block formatted as multi-line rather than inlined.  If there are cases where forcing multi-line block formatting is desired the current solution would be to add a line comment within the block.